pages/working-with-projects/getting-help-while-you-work.md

```diff
@@ -13,3 +13,9 @@ In addition to this online help file, there are several additional help resource
 To activate context help, press F1. This would open the help section at the right side of the AppGini window. This section displays help on whatever element has the focus at the moment. You can use the mouse or the Tab key to move the focus between all the elements of the project window. To hide the help section, press F1 again.
 
 
+![AppGini_BtmGdfWFH4](https://github.com/bigprof-software/appgini-docs/assets/168858457/1b8c5ff8-8f8a-4a5e-9647-6ba752df8948)
+
+
+in the right side pannel by pressing F1 key it displays help on whatever element has the focus at the moment.
```

Comment:
Please make sure the punctuation and grammar are revised. I'd suggest using ChatGPT or Bard for that purpose. You could use a prompt like:

Please revise the following text for grammar, punctuation and clarity:

```
[INSERT PAGE CODE HERE]
```